operators/src/engine/query_processor.rs

```diff
@@ -222,6 +222,63 @@ impl TypedRasterQueryProcessor {
     }
 }
 
+impl From<Box<dyn RasterQueryProcessor<RasterType = u8>>> for TypedRasterQueryProcessor {
```

Comment:
where is this used?

Reply:
here: https://github.com/geo-engine/geoengine/blob/bc077e96e2f6a624b0941d6f3dc7c0962d760fa9/operators/src/processing/temporal_raster_aggregation.rs#L94